src/shared/utils/billing.ts

```diff
@@ -34,24 +30,10 @@ export const BillingRate = {
   ANNUALLY: 'annually',
 } as const
 
-export interface Plan {
```

Comment:
The source of truth for this schema is now the type in services/account

Reply:
BIG